app-web/halloy/autobuild/beyond

```diff
@@ -0,0 +1,7 @@
+abinfo "Installing desktop entry ..."
+install -Dm644 "$SRCDIR/assets/linux/org.squidowl.halloy.appdata.xml" -t "$PKGDIR/usr/share/appdata"
+install -Dm644 "$SRCDIR/assets/linux/org.squidowl.halloy.desktop" -t "$PKGDIR/usr/share/applications"
+
+abinfo "Installing icons ..."
+mkdir -p "$PKGDIR/usr/share/icons/hicolor"
```

Comment:
Strange that you created the `hicolor` subdirectory but then copied the whole `icons` directory into `/usr/share`, please double check.